README.markdown

```diff
@@ -15,10 +15,30 @@ The first step to using this repo is to create a Python virtual environment and
 
 After the first time, you just need to run the `source .venv/bin/activate` each time you want to activate it in a new shell.
 
-## Downloading all required files
+## Testing
 
-You then need to download various files from the internet, as [documented here](data/README.markdown).
+To run the test suite, from the root of the repo, and from your activated environment, run
 
-## Building the tree
+    python -m pytest
 
-Now you are set up to actually build the tree and other backend files, by following [these instructions](oz_tree_build/README.markdown)
\ No newline at end of file
+## Building the latest tree from OpenTree
+
+### Setup
+
+We assume that you want to build a OneZoom tree based on the most recent online OpenTree version.
+You can check the most recent version of both the synthetic tree (`synth_id`) and the taxonomy (`taxonomy_version`) via the
+[API](https://github.com/OpenTreeOfLife/germinator/wiki/Open-Tree-of-Life-Web-APIs) e.g. by running `curl -X POST https://api.opentreeoflife.org/v3/tree_of_life/about`. Later in the build, we use specific environment variables set to these version numbers. Assuming you are in a bash shell or similar, you can set them as follows:
+
+```
+OT_VERSION=14_7 #or whatever your OpenTree version is
+OT_TAXONOMY_VERSION=3.5
+OT_TAXONOMY_EXTRA=draft1 #optional - the draft for this version, e.g. `draft1` if the taxonomy_version is 3.5draft1
+```
```

Comment:
Right now, I have these defined in the tree building instructions page, since that's where they end up being needed. Do we really need them here as well? The challenge is to then have to change them in both places when we update the versions.

Reply:
#14 removes them from there.